### PR TITLE
Add caption to my 24-uskaxg74.md

### PR DESCRIPTION
**Pull request by uskaxg74**
Create pull request.

### CODE REVIEW

File: _slides/24-uskaxg74.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (mikechappelow) [inline _slides/24-uskaxg74.md:6]: reviewed comment change

**Review by mikechappelow — APPROVED** [review]
Summary of review comments here!!!

**Review by uskaxg74 — COMMENTED** [review]
Test comment.

**Comment by uskaxg74** [comment]
Merging.

**Pull request closed without merging**